packages/react-instantsearch-dom-maps/src/GoogleMaps.js

```diff
@@ -145,16 +141,17 @@ class GoogleMaps extends Component {
   }
 
   render() {
-    const { children } = this.props;
+    const { google, children } = this.props;
     const { isMapReady } = this.state;
 
     return (
-      <div className={cx('')}>
-        <div ref={c => (this.element = c)} className={cx('map')} />
-        {isMapReady && children}
-      </div>
+      <GoogleMapsContext.Provider value={{ google, instance: this.instance }}>
```

Comment:
We should not use inline value here, it will re-render the `Consumer` on each render. We have [to always provide the same reference](https://reactjs.org/docs/context.html#caveats) unless the state changes.